src/scripts/main.js

```diff
@@ -1,3 +1,12 @@
 'use strict';
 
-// write your code here
+const mainTable = document.querySelector('table');
+const allRows = [...mainTable.rows];
+
+for (let i = 0; i < allRows.length; i++) {
+  const cells = [...allRows[i].cells];
+
+  const copy = cells[1].cloneNode(true);
```

Comment:
Ensure that each row has at least two cells to avoid errors when accessing `cells[1]`. If a row has fewer than two cells, this line will throw an error.